perf-tests/clusterloader2/pkg/measurement/common/bundle/test_metrics.go

```diff
@@ -41,8 +43,12 @@ func createTestMetricsMeasurement() measurement.Measurement {
 	if metrics.etcdMetrics, err = measurement.CreateMeasurement("EtcdMetrics"); err != nil {
 		klog.Errorf("%v: etcdMetrics creation error: %v", metrics, err)
 	}
-	if metrics.schedulingMetrics, err = measurement.CreateMeasurement("SchedulingMetrics"); err != nil {
-		klog.Errorf("%v: schedulingMetrics creation error: %v", metrics, err)
+	// We temporarily disable the metric collection on scheduler in scale-out scenarios,
```

Comment:
what's the reason to disable?

Reply:
I guess this is pending design on how to gather and summarize scheduler metric with multiple TPs ( tenants on each TP ). there should be at least issues to track this task --- it is a major change in scale out design to perf test metric that we need to support.

Reply:
In scale-out clusters, The scheduler component is in a different partition from the resource partition. So the perf-test tool complains about failures  to get the scheduler pod. 

Changes are needed to enable scheduler metrics collection in scale-out perf tests. 

I opened https://github.com/CentaurusInfra/arktos/issues/930 to track this work.